platform/android/MapLibreAndroid/src/main/java/org/maplibre/android/http/LocalRequest.kt

```diff
@@ -0,0 +1,49 @@
+package org.maplibre.android.http
+
+import android.content.res.AssetManager
+import org.maplibre.android.MapLibre
+import org.maplibre.android.MapStrictMode
+import org.maplibre.android.log.Logger
+import java.io.IOException
+import java.io.InputStream
+
+fun localRequest(url: String, onCompletion: ((ByteArray?) -> Unit)?) {
```

Comment:
Does this need to be public?

Reply:
No.  It's only ever used in the same package.

Reply:
Sorry, I misunderstood your question.  Fixed.